scripts/install.sh

```diff
@@ -0,0 +1,226 @@
+,------.   ,--.              ,--.                    
+|  .---' ,-|  | ,---.  ,---. |  |-.  ,---.,--.  ,--. 
+|  `--, ' .-. || .-. || .-. :| .-. '| .-. |\  `'  /  
+|  `---.\ `-' |' '-' '\   --.| `-' |' '-' '/  /.  \  
+`------' `---' .`-  /  `----' `---'  `---''--'  '--' 
+               `---'                                 
+EOF
+
+# Default Values
+DEFAULT_HOSTNAME="edgebox"
+DEFAULT_TIMEZONE_COUNTRY_CODE="DE"
+DEFAULT_KEYBOARD_LAYOUT_COUNTRY_CODE="de"
+DEFAULT_LOCALE="en_US"
+RASPBERRYPI_UA_NETINST_RELEASE="v2.4.0_caf7423"
+RASPBERRYPI_UA_NETINST_RELEASE_URL="https://github.com/FooDeas/raspberrypi-ua-netinst/releases/download/v2.4.0_caf7423/raspberrypi-ua-netinst-git-caf7423.zip"
+
+# Color Escape Codes
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+YELLOW='\033[0;33m'
+BLUE='\033[0;34m'
+NC='\033[0m'
+
+# File Paths
+USER_INSTALLER_CONFIG_LOCATION=./image/raspberrypi-ua-netinst/config/installer-config.txt
+
+# Functions
+function print_success {
+    # Prints a message in green
+    echo -e "${GREEN}$1${NC}"
+}
+
+function print_error {
+    # Print a message in red
+    echo -e "${RED}$1${NC}"
+}
+
+function print_warning {
+    # Prints a message in yellow
+    echo -e "${YELLOW}$1${NC}"
+}
+
+function print_question {
+    # Prints a message in blue
+    echo -e "${BLUE}$1${NC}"
+}
+
+function get_yes_or_no() {
+    # Ask for a yes or no answer
+    # Keep attempting to read until a valid answer is given
+    local response
+    read response
+    while [[ ! $response =~ ^[yn]$ ]]; do
+        echo "Please enter y or n"
+        read response
+    done
+    echo $response
+}
+
+# Print a message
+echo "This process will ask you a couple of questions to setup..."
+echo "Press [ENTER] to continue..."
+
+# Wait for user input
+read
+
+# Ask for the root password
+print_question "Type the root password for the system:"
+read -s root_password
+echo
+
+# Ask if root user should have SSH access
+print_question "Should the root user have SSH access? (y/n):"
+root_ssh=$(get_yes_or_no)
+echo
+
+
+# Convert answer to 0 if n, 1 of y
+if [[ $root_ssh == "n" ]]; then
+    root_ssh=0
+else
+    root_ssh=1
+fi
+
+# Ask for the user password
+print_question "Please enter the password for the system user (sudo user):"
+read -s user_password
+echo
+
+# Ask if user is going to use Wifi to connect to the network
+print_question "Will you use Wifi to connect to the network? (y/n):"
+wifi=$(get_yes_or_no)
+echo
+
+# If user is going to use Wifi, ask for the SSID and password
+if [[ $wifi == "y" ]]; then
+    print_question "Type the SSID of the network:"
+    read ssid
+    echo
+
+    print_question "Type the password of the network:"
+    read -s wifi_password
+    echo
+
+    print_question "Type the country code of the network (eg. DE, PT, ES, FR):"
+    read wifi_country_code
+    echo
+fi
+
+# As if user wants to introduce a custom timezone country code
+print_question "Do you want to introduce a custom timezone country code? (y/n):"
```

Comment:
Should we output what the default one is for these questions?

Reply:
Yes that is a good idea. Adding that